config/default.yml

```diff
@@ -743,7 +743,11 @@ Naming/UncommunicativeMethodArgName:
   MinNameLength: 3
   AllowNamesEndingInNumbers: true
   # Whitelisted names that will not register an offense
-  AllowedNames: []
+  AllowedNames:
+    - io
+    - to
+    - id
+    - '*'
```

Comment:
`'*'` does not suppress `def foo(*)`. Because `*` is not a variable name.

```bash
$ ruby-parse -e def foo(*) end
(def :foo
  (args
    (restarg)) nil)

$ ruby-parse -e def foo(*a) end
(def :foo
  (args
    (restarg :a)) nil)
```

So, I think we should add `AllowUnnamedRestArgument` option or something, or ignore unnamed rest argument always.

Reply:
Thanks!  I'll remove it and leave the new config for a separate PR.